hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestEndpoints.java

```diff
@@ -133,11 +135,14 @@
  * Test for Recon API endpoints.
  */
 public class TestEndpoints extends AbstractReconSqlDBTest {
-  private NodeEndpoint nodeEndpoint;
-  private PipelineEndpoint pipelineEndpoint;
-  private ClusterStateEndpoint clusterStateEndpoint;
-  private UtilizationEndpoint utilizationEndpoint;
-  private MetricsProxyEndpoint metricsProxyEndpoint;
+  private org.apache.hadoop.ozone.recon.api.NodeEndpoint nodeEndpoint;
+  private org.apache.hadoop.ozone.recon.api.PipelineEndpoint pipelineEndpoint;
+  private org.apache.hadoop.ozone.recon.api.ClusterStateEndpoint
+      clusterStateEndpoint;
+  private org.apache.hadoop.ozone.recon.api.UtilizationEndpoint
+      utilizationEndpoint;
+  private org.apache.hadoop.ozone.recon.api.MetricsProxyEndpoint
+      metricsProxyEndpoint;
```

Comment:
Adding `org.apache.hadoop.ozone.recon.api.` seems unnecessary (also elsewhere in the same file).